### PR TITLE
Modification in some functionality and solved some minor bugs.

### PR DESCRIPTION
**Pull request by Click2cloud-Eros**
**What type of PR is this?**
1) bug fix
2) enhancement

**What this PR does / why we need it?**
This PR included some modification and bugs fixes.

**Which modification and issue(s) this PR fixes:** 
1) Merge some array attributes (i.e. titles, drawables, values, percents, colors) into single ArrayList attribute (i.e. NodeList).
2) Add **enabledMultiColorRegion** and **enabledCircularOutline** attribute for xml.
3) Solved last button cut issue.
4) Solved setCount issue.
5) Change functionality of **Load Animation** button.
6) Add and remove Junit testcases as per new implementation.
7) Update readme file content as per new implementation.
8) Move Attributes to new file.

### CODE REVIEW

File: xradar/src/main/java/com/orzangleli/radar/XRadarView.java [hunk 1]
```diff
@@ -24,53 +21,52 @@
...
     private int descPadding = 5;
+
+    // Title size
     private int titleSize = 40;
 
+    // The ratio of the radar chart graphic to the entire space
     private float radarPercent = 0.7f;
 
+    // When the gradient is turned on, the color at the center of the circle
     private Color startColor = new Color(RgbPalette.parse("#80FF0000"));
```
Comment (kanaksony) [inline xradar/src/main/java/com/orzangleli/radar/XRadarView.java]: Please move this and similar in resources

**Review by kanaksony — CHANGES_REQUESTED** [review]
Please add Copyright for new code files.
Add colors code in resources instead of code.
Utils.java# getPathById() shall be optimized like - 
` 
public static String getPathById(Context context, int id) {
        if (context != null) {
            ResourceManager manager = context.getResourceManager();
            if (manager != null) {
                try {
                    return manager.getMediaPath(id);
                } catch (IOException | NotExistException | WrongTypeException e) {
                    //TO-DO: Handle exception here
                    return "";
                }
            }
        }
        return "";
    }
`

Node - There are some methods never called

**Comment by Click2cloud-Eros** [comment]
> Please add Copyright for new code files. Add colors code in resources instead of code. Utils.java# getPathById() shall be optimized like - ` public static String getPathById(Context context, int id) { if (context != null) { ResourceManager manager = context.getResourceManager(); if (manager != null) { try { return manager.getMediaPath(id); } catch (IOException | NotExistException | WrongTypeException e) { //TO-DO: Handle exception here return ""; } } } return ""; }`
> 
> Node - There are some methods never called

I seen, I have already added copyright in new files.
And other changes I will do.

**Review by kanaksony — APPROVED** [review]
Looks good

**Review by kanaksony — APPROVED** [review]

**Review by sharathks84 — APPROVED** [review]
Okay Approved

**Review by applibengineering — APPROVED** [review]

**Pull request merged**